### PR TITLE
1.3b order complete

### LINKED ISSUE #112: As a restaurant owner I want to indicate that an order has been completed
... so that I can see which orders there are left to be completed

### PR DESCRIPTION
**Pull request by IanCKW**
Add order complete [index] command and order list only shows uncompleted orders now

### CODE REVIEW

**Review by jamesleeht — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/order/OrderDeleteCommand.java [hunk 1]
```diff
@@ -37,8 +37,8 @@ public OrderDeleteCommand(Index targetIndex) {
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
 
-        List<Order> lastShownList = model.getFilteredOrderList();
-
+        List<Order> lastShownList = model.getFilteredOrderList(Order.State.UNCOMPLETED);
+        System.out.println(targetIndex.getZeroBased());
```
Comment (Sharptail) [inline src/main/java/seedu/address/logic/commands/order/OrderDeleteCommand.java]: Is this printing intentional? If it is, should use logging instead :)

**Review by Sharptail — COMMENTED** [review]
One minor code quality issue found, please fix before merging :D

**Review by Sharptail — APPROVED** [review]
LGTM!

**Pull request merged**